http://sourceware.org/ml/gdb-cvs/2012-04/msg00183.html

### src/gdb/doc/ChangeLog	2012/04/22 15:49:21	1.1299
### src/gdb/doc/ChangeLog	2012/04/23 17:20:56	1.1300
## -1,3 +1,8 @@
+2012-04-23  Jan Kratochvil  <jan.kratochvil@redhat.com>
+
+	* gdb.texinfo (Auto-loading safe path): Remove trailing [@dots{}].
+	Three times.
+
 2012-04-22  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	* gdb.texinfo (Auto-loading safe path): Replace @itemize @bullet
--- src/gdb/doc/gdb.texinfo	2012/04/22 15:49:21	1.946
+++ src/gdb/doc/gdb.texinfo	2012/04/23 17:20:56	1.947
@@ -21142,11 +21142,11 @@
 You have to specify also any existing directories displayed by
 by @samp{show auto-load safe-path} (such as @samp{/usr:/bin} in this example).
 
-@item @kbd{gdb -iex "set auto-load safe-path /usr:/bin:~/src/gdb" [@dots{}]}
+@item @kbd{gdb -iex "set auto-load safe-path /usr:/bin:~/src/gdb"}
 Specify this directory as in the previous case but just for a single
 @value{GDBN} session.
 
-@item @kbd{gdb -iex "set auto-load safe-path" [@dots{}]}
+@item @kbd{gdb -iex "set auto-load safe-path"}
 Disable auto-loading safety for a single @value{GDBN} session.
 This assumes all the files you debug during this @value{GDBN} session will come
 from trusted sources.
@@ -21161,7 +21161,7 @@
 also suppresses any such warning messages:
 
 @table @asis
-@item @kbd{gdb -iex "set auto-load no" [@dots{}]}
+@item @kbd{gdb -iex "set auto-load no"}
 You can use @value{GDBN} command-line option for a single @value{GDBN} session.
 
 @item @file{~/.gdbinit}: @samp{set auto-load no}
